AAI/README.md

```diff
@@ -76,7 +71,7 @@ of a full-broker. This is one possible way to use this spec.
 In this diagram, the Data Owner Claim Clearinghouse, the Data Holder Claim
 Clearinghouse and the Broker are all different entities. However, in
 many cases, we expect the Broker and Data Owner to be the same entity and
```

Comment:
This is a strong expectation. In ELIXIR, data owners (e.g. EGA) are separate from the broker (ELIXIR AAI). I believe it is the same for Google broker.